docs/README.md

```diff
@@ -2,3 +2,60 @@
 
 This site uses [Hugo](https://gohugo.io/) with the
 [Docsy](https://www.docsy.dev/docs/) theme.
+
+## Local Build
+In order to build the website locally, you will need to install `hugo` and `npm`.
+For example, on Ubuntu: `sudo apt-get install hugo npm`
+
+Note that `.github/workflows/docs.yml` installs a specific version of Hugo,
+and it is likely that your system-provided version is not exactly the same.
+See the bottom of this README for instruction on how to install the exact version.
+
+
+1. Build the markdown files from the tablegen sources:
+    ```bash
+    bazel query "filter('_filegroup', siblings(kind('gentbl_rule', @heir//...)))" | xargs bazel build "$@"
+    ```
+
+1. Copy the markdown files to `docs/`:
+    ```bash
+    ./.github/workflows/copy_tblgen_files.sh
+    ```
+
+1. Navigate to the `/docs` directory:
+    ```bash
+    cd docs
+    ```
+
+1. Use `npm` to install the dependencies:
+    ```bash
+    npm install
+    ```
+
+1. Run `hugo`:
+    ```bash
+    hugo --minify
```

Comment:
Probably better to run `hugo --server --minify` to get a locally-served version one can inspect.

Reply:
Done